catalog/ui/catalog-ui-search/src/main/webapp/component/list-item/list-item.view.js

```diff
@@ -24,6 +24,9 @@ var DropdownView = require('component/dropdown/popout/dropdown.popout.view');
 var ListEditorView = require('component/list-editor/list-editor.view');
 var QueryFeedView = require('component/query-feed/query-feed.view');
 var ListInteractionsView = require('component/list-interactions/list-interactions.view');
+var lightboxInstance = require('component/lightbox/lightbox.view.instance');
+var IngestDetailsView = require('component/ingest-details/ingest-details.view');
+var store = require('js/store');
```

Comment:
Let's remove the store reference and use the list directly since we have it available here.

Reply:
fixed